.github/workflows/on_pr.yml

```diff
@@ -119,3 +119,13 @@ jobs:
               with:
                   source-dir: docs/out
                   action: remove
+
+    status-check:
```

Comment:
hm it seems like even with this we are hanging on a status check on this PR? 
![image](https://github.com/user-attachments/assets/7cfb1a4b-a4b8-46a7-8815-982cf0e31878)

although this PR doesn't have the preview tag, so not sure what's going on

Reply:
yeah this is due to our branch protection being configured to look for "CI", but now CI is split up into 4 (soon to be 5) checks which are different, so it's not recognizing it. If this is merged I want to change the check to look for "status-check" instead, which should solve this issue!